lib/generators/rolemodel/modals/README.md

```diff
@@ -2,31 +2,33 @@
 
 ## What you get
 
-* Custom confirms
-* Modal and panel styling
-* Modal and panel view and link helper
-* Modal and panel javascript
+* Custom confirm dialogs via `@rolemodel/turbo-confirm` integration
+* Modal and panel styling ..via `@rolemodel/optics` integration?
+* Modal and panel layouts w/ link helpers (or you could just, e.g. `link_to ... data: {turbo_frame: 'modal'}`)
+* frame-missing handler (required for turbo-rails v1.4.0 and above)
 
 Adds UI and JavaScript for opening modals and the panel within a Rails view
 
 ## Example delete confirmation link
-`= link_to "Test Confirm", delete_path(model), class: 'btn btn--primary', data: { confirm: 'Are you sure?', confirm_details: "This will delete #{model.name}. This action cannot be undone.", confirm_button: 'Yes, delete it' }`
+`= link_to "Test Confirm", delete_path(model), class: 'btn btn--primary', data: { turbo_confirm: 'Are you sure?', confirm_details: "This will delete #{model.name}. This action cannot be undone." }`
 
 The data attributes needed to make this work
-- `confirm` (required) - Just like rails basic confirm functionality. This will be header text. E.g. "Are you sure?"
+- `turbo_confirm` (required) - Just like rails basic confirm functionality. This will be header text. E.g. "Are you sure?"
 - `confirm_details` (optional, default: nil) - Provides additional details/text to the action the user about to take. E.g. "This will remove everything"
-- `confirm_button` (optional, default: "Yes, I'm sure") - Allows the button text affirming the action to be changed
+- additional content slots need to be configured. See @rolemodel/turbo-confirm on Github for details.
```

Comment:
perhaps @rolemodel/turbo-confirm should be updated to support `confirm_button` as a default content-slot?

Reply:
this is done